cloudflare/data_source_zones_test.go

```diff
@@ -21,8 +19,15 @@ func init() {
 }
 
 func testSweepCloudflareZones(r string) error {
-	client, _ := cloudflare.New(os.Getenv("CLOUDFLARE_TOKEN"), os.Getenv("CLOUDFLARE_EMAIL"))
-	zones, _ := client.ListZones("baa.com", "baa.net", "baa.org", "foo.net")
+	client, clientErr := sharedClient()
+	if clientErr != nil {
+		log.Fatalf("[ERROR] Failed to create Cloudflare client: %s", clientErr)
+	}
+
+	zones, zoneErr := client.ListZones("baa.com", "baa.net", "baa.org", "foo.net")
+	if zoneErr != nil {
+		log.Fatalf("[ERROR] Failed to fetch Cloudflare zones: %s", zoneErr)
```

Comment:
Logging via `Fatal*` also has a "side effect" of `os.Exit(1)` which I don't think is necessary here? We can achieve pretty much the same by just returning the error, which also gives us the opportunity to perform any cleanup higher up the call stack.

Reply:
Fair enough 👍